sdk/instance/instance.go

```diff
@@ -65,7 +65,7 @@ func (i *Instance) Create(id string, env []string) (*instance.Instance, error) {
 		return nil, err
 	}
 
-	// overwrite default env vars with user defined ones.
+	// calculate env vars by overwriting user defined one's with defaults.
 	instanceEnv := xos.EnvMergeMaps(xos.EnvSliceToMap(srv.Configuration.Env), xos.EnvSliceToMap(env))
```

Comment:
I chose to not optimise this in this PR since it's not the subject of this one, and I believe that env vars type should be changed to a map instead a slice so there might be some small refactoring coming in future.